p2d/p2d.py

```diff
@@ -114,55 +119,59 @@ def get_checker_md5(checker):
             if interactor is not None:
                 logger.info('Use custom interactor.')
                 with open(yaml_file, 'w', encoding='utf-8') as f:
+                    f.write(yaml_content)
                     f.write('validation: custom interactive\n')
                 ensure_dir(interactor_dir)
-                copyfile(testlib, '{}/testlib.h'.format(interactor_dir))
-                interactor_file = '{}/{}'.format(package_dir, interactor.find('source').attrib['path'])
-                copyfile(interactor_file, '{}/interactor.cpp'.format(interactor_dir))
+                copyfile(testlib, f'{interactor_dir}/testlib.h', follow_symlinks=False)
```

Comment:
![image](https://user-images.githubusercontent.com/10959348/121523798-306f4300-ca29-11eb-9c15-0d1868edcf21.png)

When you are not in the p2d directory, `../testlib.h` may not exist. So copyfile is failed and the warning is ignored. I think the symbolic link is not acceptable.